lib/config/options/index.ts

```diff
@@ -124,6 +124,25 @@ const options: RenovateOptions[] = [
     default: [],
     globalOnly: true,
   },
+  {
+    name: 'bumpVersions',
+    description:
+      'A list of bumpVersion config options to bump generic version numbers.',
+    type: 'array',
+    subType: 'object',
+    default: [],
+    cli: false,
+    env: false,
+    experimental: true,
+  },
+  {
+    name: 'bumpType',
+    description:
+      'The semver level to use when bumping versions. This is used by the `bumpVersions` feature.',
+    type: 'string',
+    default: 'patch',
```

Comment:
We're missing the the `allowedValues`: https://github.com/npm/node-semver/blob/30c438bb46c74f319aa8783f96d233ebf5f4a90d/internal/constants.js#L16-L24